sklearnex/covariance/incremental_covariance.py

```diff
@@ -196,6 +198,45 @@ def _onedal_partial_fit(self, X, queue=None, check_input=True):
 
         return self
 
+    @wrap_output_data
+    def score(self, X_test, y=None):
```

Comment:
```suggestion
    def score(self, X, y=None):
```

Reply:
I agree, but it was done for conformance reasons: https://github.com/scikit-learn/scikit-learn/blob/main/sklearn/covariance/_empirical_covariance.py#L255

Reply:
Oh, yeah. Good catch. I thought, exactly for the conformance it should be `X`. Thank you for providing the link